Documentation/configuration/api-restrictions-table.rst

```diff
@@ -128,6 +128,8 @@ The following API flags are compatible with the ``cilium-operator`` flag
 ===================== ====================
 Flag Name             Description
 ===================== ====================
+GetCluster            This path will return the list of remote clusters and their
+                      status.
```

Comment:
If you're also ready to edit `GetHealthz`'s description for consistency, it would read better as:

```suggestion
GetCluster            Returns the list of remote clusters and their status.
```

(non-blocking)

Reply:
Updated, thanks! While being there' I've also added a description for the metrics path.